DEVEL.md

```diff
@@ -207,16 +207,19 @@ Fork the [Community Operators](https://github.com/k8s-operatorhub/community-oper
 $ git clone https://github.com/<GitHub Username>/community-operators
 ```
 
-Generate package manifests with:
+Generate bundle and build bundle image:
 ```
-$ make packagemanifests OPERATOR_VERSION=0.X.Y
+$ make bundle OPERATOR_VERSION=0.X.Y
+$ make bundle-build
 ```
 
+> **Note**: You need to pull the image to a registry if you want to follow the verification process below.
```

Comment:
```suggestion
> **Note**: You need to push the image to a registry if you want to follow the verification process below.
```

Reply:
Oops, thanks!

Reply:
@hj-johannes-lee can you accept suggested change?

Reply:
@bart0sh Oops, I think I pushed without changing last time. I pushed just now again.! Thanks!